.github/workflows/build-wheels.yml

```diff
@@ -216,23 +216,6 @@ jobs:
           python wheel/generate_type_stubs.py
           git diff --exit-code
 
-  build-tools:
-    name: build chia-tools
-    runs-on: ubuntu-latest
-    steps:
-      - name: Clean workspace
-        uses: Chia-Network/actions/clean-workspace@main
-
-      - name: Checkout code
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-
-      - name: cargo build
-        run: |
-          cd crates/chia-tools
-          cargo build
```

Comment:
I don't think `cargo test` will build these targets.

Reply:
Added `cargo build --workspace --all-features --release`, which will cover it